2020-06-18 09:11:51,366:Survey Assist - v1.0 :************************** STARTED APPLICATION - User: ??**************************
2020-06-18 09:12:00,694:Survey Assist - v1.0 :OPENING UP A GSI FILE: C:/Users/Richard/PycharmProjects/Survey Assist/Test Files/GSI Files/2D Survey MR050819.GSI
2020-06-18 09:12:07,865:Survey Assist - v1.0 :************************** STARTED APPLICATION - User: ??**************************
2020-06-18 09:12:10,896:Survey Assist - v1.0 :OPENING UP A GSI FILE: C:/Users/Richard/PycharmProjects/Survey Assist/Test Files/GSI Files/ARTC_903_TS15 230320.GSI
2020-06-18 09:12:45,859:Survey Assist - v1.0 :************************* STARTED APPLICATION - User: ?? *************************
2020-06-18 09:12:50,415:Survey Assist - v1.0 :OPENING UP A GSI FILE: C:/Users/Richard/PycharmProjects/Survey Assist/Test Files/GSI Files/3D STN named incorrect.GSI
2020-06-22 11:22:34,480:Survey Assist - v1.0 :************************* STARTED APPLICATION - User: RW *************************
2020-06-23 09:43:53,374:Survey Assist - v1.0 :************************* STARTED APPLICATION - User: RW *************************
2020-06-23 12:45:00,723:Survey Assist - v1.0 :************************* STARTED APPLICATION - User: RW *************************
2020-06-23 12:45:19,997:Survey Assist - v1.0 :OPENING UP A GSI FILE: C:/Users/rjwal_000/PycharmProjects/Survey Assist/Test Files/GSI Files/2D Survey MR050819.GSI
2020-06-23 12:46:56,442:Survey Assist - v1.0 :************************* STARTED APPLICATION - User: RW *************************
2020-06-24 09:52:59,081:Survey Assist - v1.0 :Job Tracker excel spreadsheet not found

[Errno 2] No such file or directory: 'C:\\Users\\rjwal_000\\PycharmProjects\\Survey Assist\\Test Files\\Survey Data\\2020\\Job Tracker Digital System.xlsm'
Traceback (most recent call last):
  File "C:\Users\rjwal_000\PycharmProjects\Survey Assist\job_tracker.py", line 19, in __init__
    self.workbook = load_workbook(self.job_tracker_filepath, read_only=True)
  File "C:\Users\rjwal_000\AppData\Local\Programs\Python\Python38-32\lib\site-packages\openpyxl\reader\excel.py", line 312, in load_workbook
    reader = ExcelReader(filename, read_only, keep_vba,
  File "C:\Users\rjwal_000\AppData\Local\Programs\Python\Python38-32\lib\site-packages\openpyxl\reader\excel.py", line 124, in __init__
    self.archive = _validate_archive(fn)
  File "C:\Users\rjwal_000\AppData\Local\Programs\Python\Python38-32\lib\site-packages\openpyxl\reader\excel.py", line 96, in _validate_archive
    archive = ZipFile(filename, 'r')
  File "C:\Users\rjwal_000\AppData\Local\Programs\Python\Python38-32\lib\zipfile.py", line 1250, in __init__
    self.fp = io.open(file, filemode)
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\rjwal_000\\PycharmProjects\\Survey Assist\\Test Files\\Survey Data\\2020\\Job Tracker Digital System.xlsm'
2020-06-24 09:53:05,680:Survey Assist - v1.0 :Error has occurred creating Job Tracker object.

No Job Names Found
Traceback (most recent call last):
  File "C:/Users/rjwal_000/PycharmProjects/Survey Assist/main.py", line 4400, in __init__
    self.job_tracker_bar = JobTrackerBar(self.main_window, self.menu_bar.user_config.user_initials)
  File "C:/Users/rjwal_000/PycharmProjects/Survey Assist/main.py", line 2109, in __init__
    self.jt_job_name_combo['values'] = self.get_combobox_values()
  File "C:/Users/rjwal_000/PycharmProjects/Survey Assist/main.py", line 2195, in get_combobox_values
    raise Exception("No Job Names Found")
Exception: No Job Names Found
2020-06-24 09:53:42,733:Survey Assist - v1.0 :************************* STARTED APPLICATION - User: RW *************************
2020-06-24 10:02:40,195:Survey Assist - v1.0 :************************* STARTED APPLICATION - User: RW *************************
2020-06-24 10:04:26,210:Survey Assist - v1.0 :OPENING UP A GSI FILE: C:/SD/TS60/Gsi/A9MR210120.GSI
2020-06-24 10:08:28,111:Survey Assist - v1.0 :************************* STARTED APPLICATION - User: RW *************************
2020-06-24 10:08:32,665:Survey Assist - v1.0 :OPENING UP A GSI FILE: C:/SD/TS60/Gsi/A9MR210120.GSI
